CHANGELOG.next.asciidoc

```diff
@@ -617,6 +617,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Ensure common proxy settings support in HTTP clients: proxy_disabled, proxy_url, proxy_headers and typical environment variables HTTP_PROXY, HTTPS_PROXY, NOPROXY. {pull}25219[25219]
 - `add_process_metadata` processor enrich process information with owner name and id. {issue}21068[21068] {pull}21111[21111]
 - Add proxy support for AWS functions. {pull}26832[26832]
+- Add max_content_length as a mechanism to throttle large prometheus endpoints {pull}27087[27087]
```

Comment:
Update this if needed according to https://github.com/elastic/beats/pull/27087/files#r678074047